src/igvfd/schemas/changelogs/prediction_set.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog for *`prediction_set.json`*
 
+### Schema version 4
+
+* Rename `targeted_genes` to `genes`.
+* Rename `targeted_loci` to `loci`.
+* Add `small_scale_gene_list`, `large_scale_gene_list`, `small_scale_loci_list`, and `large_scale_loci_list`.
+* Restrict `genes` and `loci` to submittable by admins only. Please submit `small_scale_gene_list`, `large_scale_gene_list`, `small_scale_loci_list`, or `large_scale_loci_list` instead.
```

Comment:
remove the part "Please submit..." i dont think it is a part of change log
in general the idea is to have as concise as possible statements making it possible to read through changelog

Reply:
removed